atd-vzd/migrations/default/1715960019005_crashes_list_and_fatality_metrics_views/up.sql

```diff
@@ -242,3 +242,70 @@ left join
     lookups.injry_sev_lkp
     on lookups.injry_sev_lkp.id = crash_injury_metrics_view.crash_injry_sev_id;
 
+
+create materialized view locations_list_view as (
```

Comment:
this is very closely copied from the `locations_with_crash_injury_counts` view: https://github.com/cityofaustin/atd-vz-data/blob/master/atd-vzd/views/locations_with_crash_injury_counts.sql